foursight_core/app_utils.py

```diff
@@ -282,10 +282,11 @@ def get_portal_url(self, env_name: str) -> Optional[str]:
                 portal_url = environment_and_bucket_info.get("fourfront")
                 self._cached_portal_url[env_name] = portal_url
             except Exception as e:
+                if raise_exception:
+                    raise e
```

Comment:
To re-raise the same exception, just say `raise`, not `raise e`. There is some magic in `raise` that knows you mean the exception you're handling but also less risk that there's a semantic difference. Raising an exception anew may give you a different sense of what the source point is for the stack trace, and may give it the sense you are overriding a previous exception handling rather than continuing it. `raise` with no args means "continue to raise" while with an argument it means start raising, unless they've got some special hack in there to notice the same exception object. (But that may be unreliable because you might be able to reuse an exception object. Not sure it has to be unique.)

Reply:
Ah, thank you, I did not know this, will fix.